thirdeye/thirdeye-frontend/app/pods/manage/alert/template.hbs

```diff
@@ -130,84 +160,188 @@
+                {{/each}}
+              </ul>
+            </div>
 
-                      <a class="te-alert-page-anomalies__table-link" href="http://thirdeye.corp.linkedin.com/thirdeye#anomalies?anomaliesSearchMode=id&pageNumber=1&anomalyIds={{anomaly.anomalyId}}">investigate</a>
-                   </td>
+          {{else}}
+            No anomalies to show.
+          {{/if}}
+
+            {{!-- Alert anomaly table --}}
+            <table class="te-alert-page-anomalies__table">
+              {{#if filteredAnomalies}}
+              <thead>
+                <tr class="te-alert-page-anomalies__table-row te-alert-page-anomalies__table--head">
+                   <th class="te-alert-page-anomalies__table--cell-head">
+                    <a class="te-alert-page-anomalies__table--cell-link" {{action "toggleSortDirection" "start"}}>
+                      Start/Duration (PDT)
+                      <i class="te-alert-page-anomalies__glyph glyphicon {{if sortColumnStartUp "glyphicon-menu-up" "glyphicon-menu-down"}}"></i>
+                    </a>
+                   </th>
+                   <th class="te-alert-page-anomalies__table--cell-head">Dimensions</th>
+                   <th class="te-alert-page-anomalies__table--cell-head">
+                    <a class="te-alert-page-anomalies__table--cell-link" {{action "toggleSortDirection" "score"}}>
+                      Severity Score
+                      <i class="te-alert-page-anomalies__glyph glyphicon {{if sortColumnScoreUp "glyphicon-menu-up" "glyphicon-menu-down"}}"></i>
+                    </a>
+                   </th>
+                   <th class="te-alert-page-anomalies__table--cell-head">
+                    <a class="te-alert-page-anomalies__table--cell-link" {{action "toggleSortDirection" "change"}}>
+                      Current/WoW
+                      <i class="te-alert-page-anomalies__glyph glyphicon {{if sortColumnChangeUp "glyphicon-menu-up" "glyphicon-menu-down"}}"></i>
+                    </a>
+                   </th>
+                   <th class="te-alert-page-anomalies__table--cell-head">
+                    <a class="te-alert-page-anomalies__table--cell-link" {{action "toggleSortDirection" "resolution"}}>
+                      Resolution
+                      <i class="te-alert-page-anomalies__glyph glyphicon {{if sortColumnResolutionUp "glyphicon-menu-up" "glyphicon-menu-down"}}"></i>
+                    </a>
+                 </th>
                 </tr>
-              {{/each}}
-            </tbody>
-          </table>
-        </div>
-      {{else}}
-        <div class="te-alert-page-notification">
-          No anomalies detected. Is this an anomaly?
+              </thead>
+              {{else}}
+                No anomalies to show.
+              {{/if}}
+              <tbody>
+                {{#each paginatedFilteredAnomalies as |anomaly|}}
+                  <tr class="te-alert-page-anomalies__table-row te-alert-page-anomalies__table--body">
```

Comment:
fyi, bem need not follow the dom markup. we don't need to start at the page level (te-aler-page.... ) ,

You can just set it to a any block: `anomaly-table__row` or even `table-row__body`

```
Block

Encapsulates a standalone entity that is meaningful on its own. While blocks can be nested and interact with each other, semantically they remain equal; there is no precedence or hierarchy. Holistic entities without DOM representation (such as controllers or models) can be blocks as well.
```
http://getbem.com/naming/

Reply:
yes - refactored